OpenGL/Constructs/VAO.cs

```diff
@@ -389,7 +401,23 @@ public void BindAttributes(ShaderProgram program)
 
                 Gl.EnableVertexAttribArray((uint)loc);
                 Gl.BindBuffer(vbos[i].BufferTarget, vbos[i].ID);
-                Gl.VertexAttribPointer((uint)loc, vbos[i].Size, vbos[i].PointerType, true, vbos[i].Size * SizeOfType(vbos[i].PointerType), IntPtr.Zero);
```

Comment:
vbos[i].Size * SizeOfType(vbos[i].PointerType) has been replaced with 0 in the new code.  I think, in general, the new code will work fine (0 assumed tightly packed data).  However, is there a regression here in the case where vbos[i].Size * SizeOfType does not equal the tightly packed stride of the VBO?  From what I can tell it will always return the tightly packed stride, so replacing it with 0 should be fine, but want to get another pair of eyes on it and understand the change.